esrally/metrics.py

```diff
@@ -1849,6 +1896,26 @@ def op_metrics(op_item, key, single_value=False):
                             "max": item["max"]
                         }
                     })
+            elif metric == "transform_processing_times":
+                for item in value:
+                    all_results.append({
+                        "id": item["id"],
+                        "name": "transform_processing_times",
+                        "value": {
+                            "search": item["search"],
+                            "index": item["index"],
+                            "processing": item["processing"]
```

Comment:
this doesn't look right to me, should I instead append 3 entries to `all_results` with:

```
   {
   "id": item["id"],
   "name": "transform_search_time",
   "value": {
     "single": item["search"]
   }
   {
   "id": item["id"],
   "name": "transform_index_time",
   "value": {
     "single": item["index"]
   },
   {
   "id": item["id"],
   "name": "transform_processing_time",
   "value": {
     "single": item["processing"]
   }
```